frontend/src/routes/Send.tsx

```diff
@@ -93,6 +93,14 @@ function Send() {
       await navigateForInvoice(destination)
     } else if (paymentType === PaymentType.bip21) {
       await navigateForBip21(destination)
+    } else if (paymentType === PaymentType.keysend) {
+      if (sendAll === "true") {
+        const params = objectToSearchParams<SendConfirmParams>({ destination, all: "true" })
+        navigate(`/send/confirm?${params}`)
+      } else {
+        const params = objectToSearchParams<SendConfirmParams>({ destination })
+        navigate(`/send/amount?${params}`)
```

Comment:
Doing send all for keysend is tricky and not something we currently support, can just exclude that do just use the `else` case here

Reply:
Ok! should i go ahead and remove the if statement all together or should it send some sort of UI for the user to understand they can't sendAll for keysend

Reply:
This looks good! The sendAll parameter is never exposed through the api so you should be okay.